app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,26 @@
+class Api::V1::UsersController < ApplicationController
+  def create
+    if params[:password] != params[:password_confirmation]
+      render json: {
+        error: 'password and confirmation must match', 
```

Comment:
Is this trying to throw an error upon account creation? If so, we can handle the logic for the password and password confirmation on the FE. This would allow us to avoid running an unnecessary fetch call.

Reply:
I changed this to just rely on your password validation instead